docs/docsite/rst/user_guide/collections_using.rst

```diff
@@ -42,6 +42,11 @@ Installing a collection from a git repository
 
 .. include:: ../shared_snippets/installing_collections_git_repo.txt
 
+Installing a collection from source files
+-----------------------------------------
+
+.. include:: ../shared_snippets/installing_collections_file.rst
+
```

Comment:
Can you make a similar addition to https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/galaxy/user_guide.rst? We cover this info in two places, which is why we use the shared snippets for both of them.